client/app/billing/autoRenew/actions/delete/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,9 @@
+{
+  "autorenew_service_delete_title": "Résilier à date d'échéance",
+  "autorenew_service_delete_confirm": "Le service {{ service }} <strong>sera automatiquement résilié</strong> le {{ date }}.",
+  "autorenew_service_delete_validate": "Valider",
+  "autorenew_service_delete_warn": "Les données liées à ce service seront automatiquement supprimées de manière irréversible. Vous pourrez revenir sur cette décision jusqu'à 24h avant la date d’expiration.",
+  "autorenew_service_delete_support": "Pour résilier ce service, merci de contacter notre support technique au numéro suivant : <strong>1007</strong> et <strong>appuyez directement sur 8.</strong> Merci pour votre compréhension.",
```

Comment:
This sentence will only display in France I guess ?

Reply:
Only refactored, did not go into each product logic